Site/SiteAnalyticsModule.php

```diff
@@ -627,31 +618,7 @@ protected function getGoogleAnalyticsCommand($command)
 
 	protected function getGoogleAnalytics4Command($command)
```

Comment:
This is an empty function now. We should remove it altogether and add it later if the implementation is needed

Reply:
Done.